### PR TITLE
Added "include-all" command line option ("a" shortname)…

### PR DESCRIPTION
**Pull request by johnhelmuth-st**
…to display the entire GTFS-RT message, not  just the `entity` value. Issue #5 in upstream repo.

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -88,20 +93,26 @@ const onFeedMessage = (buf) => {
...
-		const delimeter = (printAsJSON && !isLastEntity) ? ',\n' : '\n'
-		process.stdout.write(msg + delimeter)
-	}
-	if (printAsJSON) {
-		process.stdout.write(']\n')
+	const inspectOptions = {depth, colors: printWithColors};
+	if (includeAll) {
+		const msg = printAsJSON ? JSON.stringify(data) : inspect(data, inspectOptions);
+		process.stdout.write(msg);
+	} else {
```
Comment (derhuerst) [inline cli.js:101]: I think I would prefer to `return` early here, as IMHO it makes the code slightly more readable.

**Review by derhuerst — APPROVED** [review]
Thanks for your contribution! There's only one minor aspect I have a comment about.

**Comment by derhuerst** [comment]
I have tweaked the code a bit, squashed your commits, and merged them as 629d115.

Would you like to be mentioned as a [contributor in `package.json`](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#people-fields-author-contributors)? If yes, how?

**Comment by johnhelmuth-st** [comment]
> I have tweaked the code a bit, squashed your commits, and merged them as [629d115](https://github.com/derhuerst/print-gtfs-rt-cli/commit/629d1150158fcc02b2a974d449573b91d6a6c602).
> 
> Would you like to be mentioned as a [contributor in `package.json`](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#people-fields-author-contributors)? If yes, how?

Sure, I am OK with being in the contributor property of package.json.
Use 
```
John Helmuth
john.helmuth@soundtransit.org
```

**Comment by derhuerst** [comment]
Published as `print-gtfs-rt-cli@3.1.0`. 🎉

**Pull request closed without merging**